build.gradle

```
@@ -16,7 +16,7 @@ buildscript {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}")
         classpath("org.jetbrains.kotlin:kotlin-allopen:${kotlinVersion}")
         classpath("org.jetbrains.kotlin:kotlin-noarg:${kotlinVersion}")
-        classpath('com.github.ben-manes:gradle-versions-plugin:0.21.0')
+        classpath("com.github.ben-manes:gradle-versions-plugin:0.21.0")
```

I noticed that you switched from using single quotes to double quotes in this file. It was originally single quotes for everything since double quotes are only needed for templating variables. Why was this changed?

```suggestion
        classpath('com.github.ben-manes:gradle-versions-plugin:0.21.0')
```

---

I just noticed the inconsistent use of single and double quotes. Didn't occur to me that it is related to variable substitution. I can revert that change if you want.